modules/2d_inference_3d_volume.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from monai.inferers import SlidingWindowInferer
</pre><p>Maybe it's better to put all the <code>import</code>  in a notebook cell.</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>